mkdocs.yml

```diff
@@ -1,124 +1,207 @@
+          - GraphQL: "user-guides/graphql.md"
+      - Getting Started In the Web UI:
+          - "user-guides/getting-started/index.md"
+          - Creating Devices: "user-guides/getting-started/creating-devices.md"
+          - Regions: "user-guides/getting-started/regions.md"
+          - Platforms: "user-guides/getting-started/platforms.md"
+          - Tenants: "user-guides/getting-started/tenants.md"
+          - Interfaces: "user-guides/getting-started/interfaces.md"
+          - VLANS and VLAN Groups: "user-guides/getting-started/vlans-and-vlan-groups.md"
+          - IP Address Management: "user-guides/getting-started/ipam.md"
+          - The Search Bar: "user-guides/getting-started/search-bar.md"
+      - Core Functionality:
+          - Circuits: "core-functionality/circuits.md"
+          - Devices and Cabling: "core-functionality/devices.md"
+          - Device Types: "core-functionality/device-types.md"
+          - IP Address Management: "core-functionality/ipam.md"
+          - Power Tracking: "core-functionality/power.md"
+          - Secrets: "core-functionality/secrets.md"
+          - Service Mapping: "core-functionality/services.md"
+          - Sites, Locations, and Racks: "core-functionality/sites-and-racks.md"
+          - Tenancy: "core-functionality/tenancy.md"
+          - Virtualization: "core-functionality/virtualization.md"
+          - VLAN Management: "core-functionality/vlans.md"
+      - Additional Features:
+          - Caching: "additional-features/caching.md"
+          - Change Logging: "additional-features/change-logging.md"
+          - Computed Fields: "models/extras/computedfield.md"
+          - Context Data: "additional-features/config-contexts.md"
+          - Custom Fields: "models/extras/customfield.md"
+          - Custom Links: "models/extras/customlink.md"
+          - Dynamic Groups: "models/extras/dynamicgroup.md"
+          - Export Templates: "models/extras/exporttemplate.md"
+          - Git Repositories: "models/extras/gitrepository.md"
+          - Health Check: "additional-features/healthcheck.md"
+          - Jobs: "additional-features/jobs.md"
+          - Job Hooks: "models/extras/jobhook.md"
+          - Job Scheduling and Approvals: "additional-features/job-scheduling-and-approvals.md"
+          - NAPALM: "additional-features/napalm.md"
+          - Notes: "models/extras/note.md"
+          - Prometheus Metrics: "additional-features/prometheus-metrics.md"
+          - Relationships: "models/extras/relationship.md"
+          - Statuses: "models/extras/status.md"
+          - Tags: "models/extras/tag.md"
+          - Template Filters: "additional-features/template-filters.md"
+          - Webhooks: "models/extras/webhook.md"
+      - Plugins:
+          - Using Plugins: "plugins/index.md"
+          - Developing Plugins: "plugins/development.md"
+          - Porting NetBox Plugins to Nautobot: "plugins/porting-from-netbox.md"
```

Comment:
These two probably should be moved into the `Development Guide` top-level section